LVT/Program.cs

```diff
@@ -1,12 +1,24 @@
-﻿using System;
+﻿using LVT.LVT.Services;
+using System;
+using System.IO;
 
 namespace LVT
 {
-    class Program
+    public class Program
     {
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //this takes an absolute path and needs to be adapted for each computer depending on where the file lives that you want to open - but this allows you to pass different template trees.
+            //see Tests for an alternative way that uses project resources instead - so tests can be run no matter on whose environment
+            StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\SingleBranchLVT.json");
+            JsonParser Parser = new JsonParser();
+            LeanValueTree newTree = Parser.ParseJsonLVTFromStream(file);
+            //this is an example of what is returned when you access the object:
+            //Console.WriteLine(newTree);
+            //Console.WriteLine(newTree.Vision.Title);
+            //Console.WriteLine(newTree.Vision.Goals[0]);
+            //Console.WriteLine(newTree.Vision.Goals[0].Bets[0]);
```

Comment:
Not so sure these comments should be in this file, probably better to put them in the JSONParser class if at all.

Reply:
These comments are just for demo purpose. 
The reason these comments here because -
 1. check for JSONparser class object(considering it as starting point of console app).
 2. check all other classes() objects as well.